less/wells.less

```diff
@@ -2,22 +2,39 @@
 // Wells
 // --------------------------------------------------
 
-
-// Base class
-.well {
+// Base class / mixin
+.well() {
   min-height: 24px;
-  padding: 16px;
   margin-bottom: 24px;
-  background-color: @well-bg;
   border-radius: @border-radius-base;
+
+  border: solid 1px @brand-smoke-plus-10;
+  padding: 24px;
+}
+
+.sized-well(@size) {
+  @media only screen and (min-width: @size) {
+    .well();
+  }
 }
 
 // Sizes
-.well-lg {
-  padding: 24px;
-  border-radius: @border-radius-large;
+.well-xs {
```

Comment:
I think you need to add `.well, .well-xs` for backwards compatibility

Reply:
Righto

Reply:
Espec. since i don't want to increment version major. Done though.